addons/website_sale/tests/common.py

```diff
@@ -79,3 +79,7 @@ def _create_so(self, **values):
     def _prepare_carrier(cls, product, website_published=True, **values):
         # Publish carriers by default
         return super()._prepare_carrier(product, website_published=website_published, **values)
+
+    @classmethod
+    def _create_product(cls, **kwargs):
+        return super()._create_product(website_published=True, **kwargs)
```

Comment:
also, do you think that if we need to create an unpublished product with  `website_published=False`   we do not want to override it in this case?
In other words, we can add `website_published=True`  only if it is not specified in the kwargs.

Reply:
Good idea, that will also fix the subscription tests by the way :) Thanks!